client/state/data-layer/wpcom/jetpack-onboarding/index.js

```diff
@@ -0,0 +1,59 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { get, noop } from 'lodash';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { errorNotice } from 'state/notices/actions';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { JETPACK_ONBOARDING_SETTINGS_SAVE } from 'state/action-types';
+
+/**
+ * Dispatches a request to save particular onboarding settings on a site
+ *
+ * @param   {Object} action Redux action
+ * @returns {Object} Dispatched http action
+ */
+export const saveJetpackOnboardingSettings = ( { dispatch, getState }, action ) => {
+	const { settings, siteId } = action;
+	const state = getState();
+	const token = get( state.jetpackOnboarding.credentials, [ siteId, 'token' ], null );
+	const jpUser = get( state.jetpackOnboarding.credentials, [ siteId, 'userEmail' ], null );
+
+	return dispatch(
+		http(
+			{
+				apiVersion: '1.1',
+				method: 'POST',
+				path: '/jetpack-blogs/' + siteId + '/rest-api/',
+				body: {
+					path: '/jetpack/v4/settings/',
+					body: JSON.stringify( {
+						onboarding: {
+							...settings,
+							token,
+							jpUser,
+						},
+					} ),
+					json: true,
+				},
+			},
+			action
+		)
+	);
+};
+
+export const announceSaveFailure = ( { dispatch } ) =>
+	dispatch( errorNotice( translate( 'An unexpected problem occurred. Please try again.' ) ) );
```

Comment:
Hi! I've found a possible matching string that has already been translated **26** times:
`translate( 'An unexpected error occurred. Please try again later.' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
Nailed it, @a8ci18n!